apis/nucleus/src/components/listbox/interactions/listbox-keyboard-navigation.js

```diff
@@ -49,7 +49,8 @@ export function getFieldKeyboardNavigation({ select, confirm, cancel, setScrollP
         break;
       case KEYS.ESCAPE:
         cancel();
-        return; // let it propagate to top-level
+        elementToFocus = event.currentTarget.closest('.listbox-container'); // focus jump from a cell to a listbox parent
+        break;
```

Comment:
why this change?
it looks like it should have bubbled up and be handled in listbox inline keydown before
what that broken?

Have you tested this change in ListBoxPopover case? (that closing the popover case is unchanged)